### ISSUE TITLE
可否設立候選菜單的第二字體選項

### ISSUE THREAD

**Issue by GoogleCodeExporter** [lotem/rimeime#334]
```
請寫下能夠重現此狀況的操作步驟：
1.開啓倉頡增廣字集後，試了幾款字體，都不能完整顯示包括
偏旁在內的所有字符。如能增加第二字體選項，再配合系統��
�置的默認字體，就可解決這個問題了。
2.
3.

所得的結果是這個樣： （倉頡碼 竹手 2）
預期結果該是那個樣： 

請提供以下信息，幫我們定位問題。

1. 輸入法的款式：（請選擇）中州韻 / 鼠鬚管 / 小狼毫
   輸入法版本號： 所有

2. 操作系統類型：（請選擇）Linux / Mac / Windows
   操作系統版本：所有

3. 輸入方案名稱：（如果問題涉及交互方式、輸入效果等
，請塡寫）

還想再說幾句：


```

Original issue reported on code.google.com by `kangjia...@gmail.com` on 31 Dec 2012 at 5:22

**Comment by GoogleCodeExporter** [comment]
```
字體是與平臺相關的。「小狼毫」已經利用系統提供的fallback
機制。而 Linux 的字體配置也不需要輸入法專門做。

如果首選的字體不全、第二字體就能顯示全了，那麼直接設��
�後者爲首選即可？
```

Original comment by `chen....@gmail.com` on 5 Jan 2013 at 1:00

**Comment by GoogleCodeExporter** [comment]
```
主要是想要美観。我是在蘋果系統裏，能顯示所有偏旁的字��
�似乎只有冬靑體，但其観感遠不如日文的平成字體，所以系�
��預置是平成，補了一個微軟正黑（因日文字體缺字較多，而
用微軟正黑補足最不顯突兀）。於是，鼠鬚管中若設了冬靑��
�覺得難看，設其它字體，又無法顯示所有偏旁。當然，這也�
��是大問題，只是有點美中不足。
```

Original comment by `kangjia...@gmail.com` on 5 Jan 2013 at 9:32

**Comment by GoogleCodeExporter** [comment]
```
fixed by Chongyu in Squirrel 0.9.16
```

Original comment by `chen....@gmail.com` on 21 Jan 2013 at 4:28
- Changed state: **Fixed**
- Added labels: **Type-Enhancement**, **OpSys-OSX**
- Removed labels: **Type-Defect**, **OpSys-All**
